README.md

```diff
@@ -15,52 +15,62 @@ It is used in conjunction with the `bux-wallet-backend` component, which serves
 
 There are two way of running backend locally:
 
-#### Using ./start.sh script 
+#### Using ./start.sh script
 
-`bux-wallet-frontend` provides a `start.sh` script
-which is using `docker-compose.yml` file to starts up Bux Wallet Frontend, Backend and 
-Bux Server with selected database and cache storage. To start, we need to fill the 
+`spv-wallet-web-frontend` provides a `start.sh` script
+which is using `docker-compose.yml` file to starts up `SPV Wallet` with web-frontend, web-backend and selected database and cache storage.
+
+To start, we need to fill the
 config json which we want to use, there is prepared custom config file `docker/envs/development.json`
-which is used in development environment. This script build local image of bux-wallet-frontend and use 
+which is used in development environment. This script build local image of spv-wallet-web-frontend and use
```

Comment:
Again, I forgot to update this description when changing `start.sh`

The last sentence should be just added to the previous paragraph.
like this:
```
`spv-wallet-web-frontend` provides a `start.sh` script
which is using `docker-compose.yml` file to starts up `SPV Wallet` with web-frontend, web-backend and selected database and cache storage. 
This script build local image of spv-wallet-web-frontend and use already built ones for the rest of applications.
```

(sorry for inconvenience)